.github/workflows/wheels.yml

```diff
@@ -75,10 +106,16 @@ jobs:
     # Just need to build sdist on a single machine
```

Comment:
I think you'll need to update the above to
`needs: [build_linux_wheels, build_musl_wheels, build_macosx_intel_wheels, build_macosx_arm_wheels]`